src/Config/HealthCheck/RuntimeHealthCheckConfig.cs

```diff
@@ -18,11 +18,14 @@ public record RuntimeHealthCheckConfig : HealthCheckConfig
     [JsonIgnore(Condition = JsonIgnoreCondition.Always)]
     public bool UserProvidedTtlOptions { get; init; } = false;
 
+    [JsonPropertyName("max-query-parallelism")]
+    public int MaxQueryParallelism { get; set; } = EntityCacheOptions.DEFAULT_MAX_QUERY_PARALLELISM;
+
```

Comment:
Do we need a `UserProvidedMaxQueryParallelism` flag to denote the user provided values? This way configuration doesnt bloat when writing options to the file system.. Is this property going to be configurable via the CLI?